plugins/woocommerce/tests/e2e-pw/tests/merchant/create-coupon.spec.js

```diff
@@ -25,12 +25,17 @@ test.describe( 'Add New Coupon Page', () => {
 	} );
 
 	test( 'can create new coupon', async ( { page } ) => {
-		await page.goto( 'wp-admin/post-new.php?post_type=shop_coupon' );
+		await page.goto( 'wp-admin/post-new.php?post_type=shop_coupon', {
+			waitUntil: 'networkidle',
+		} );
+
```

Comment:
In slower test sites like our nightly test site, Playwright will immediately proceeds to the next steps (typing in coupon code, etc.) even though the page hasn't finished loading yet. Waiting for `networkidle` prevents this.